README.md

```diff
@@ -6,6 +6,7 @@ Kibana is an open source ([Apache Licensed](https://github.com/elastic/kibana/bl
 
 - Elasticsearch master
 - Kibana binary package
+- 512 MB of available RAM
```

Comment:
That is correct, it's just a recommendation. We could probably run on lower, but we do have things which are not old space.